lib/spack/spack/target.py

```diff
@@ -160,3 +160,6 @@ def optimization_flags(self, compiler):
         return self.microarchitecture.optimization_flags(
             compiler.name, str(compiler_version)
         )
+
+    def is_64bit(self):
+        return '64' in self.microarchitecture
```

Comment:
`microarchitecture` is `name` from https://github.com/archspec/archspec-json/blob/master/cpu/microarchitectures.json - which doesn't always include `"64"` - we'll probably need to discuss how to detect this.

Reply:
I don't actually see this used anywhere so maybe the easiest thing to do is remove it?